packages/schemas/src/types/oidc-config.ts

```diff
@@ -7,3 +7,8 @@ export type SnakeCaseOidcConfig = {
 };
 
 export type OidcConfig = KeysToCamelCase<SnakeCaseOidcConfig>;
+
+export enum GrantType {
+  AuthorizationCode = 'authorization_code',
+  RefreshToken = 'refresh_token',
+}
```

Comment:
need to limit GrantType as partial of the OIDC Provider's grant types

Reply:
There are some related constants in oidc provider, but there are no related enums and etc.